drivers/linstorvolumemanager.py

```diff
@@ -785,6 +785,30 @@ def ensure_volume_list_is_not_locked(self, volume_uuids, timeout=None):
         if waiting:
             self._logger('No volume locked now!')
 
+    def make_volume_available(self, volume_uuid):
+        node_name = socket.gethostname()
+        volume_properties = self._get_volume_properties(volume_uuid)
+        volume_name = volume_properties.get(self.PROP_VOLUME_NAME)
+        replies = self._linstor.resource_make_available(node_name=node_name, rsc_name=volume_name)
+        if not linstor.Linstor.all_api_responses_no_error(replies):
+            raise LinstorVolumeManagerError(
+                'volume `{}` could not made available on `{}`'
+                .format(volume_uuid, node_name),
+                LinstorVolumeManagerError.ERR_GENERIC
+            )
+
+    def remove_volume_if_diskless(self, volume_uuid):
+        node_name = socket.gethostname()
+        volume_properties = self._get_volume_properties(volume_uuid)
+        volume_name = volume_properties.get(self.PROP_VOLUME_NAME)
+        replies = self._linstor.resource_delete_if_diskless(node_name=node_name, rsc_name=volume_name)
+        if not linstor.Linstor.all_api_responses_no_error(replies):
+            raise LinstorVolumeManagerError(
+                'volume `{}` could not be deleted `{}`'
+                .format(volume_uuid, node_name),
+                LinstorVolumeManagerError.ERR_GENERIC
```

Comment:
You can remove `LinstorVolumeManagerError.ERR_GENERIC`, it's already the default argument.